package.json

```diff
@@ -26,7 +26,7 @@
   "scripts": {
     "postinstall": "npm run postinstall:frontend && npm run postinstall:shared",
     "test": "npm run test:backend && npm run test:frontend",
-    "test:backend": "env-cmd -f __tests__/setup/.test-env jest --coverage --maxWorkers=4",
+    "test:backend": "env-cmd -f __tests__/setup/.test-env jest --coverage --maxWorkers=4 --testTimeout=90000",
```

Comment:
Was this needed to reduce BE test flakiness? 😮 This timeout is for individual tests, right?

Reply:
Yes, timeout is for individual test, Jest performance [declined with newer node versions ](https://github.com/jestjs/jest/issues/13904) so I increased the timeout until a fix is available

Reply:
/suggestion: there is already existing config in `jest.config.js`, suggestion to update there instead of passing in as cli arg (or passing in CLI is fine too, but we should then remove from `jest.config`)

Reply:
Sure, will move over in the second part of the MR